src/util/bitmap.h

```diff
@@ -145,6 +145,7 @@ class Bitmap {
 
   // Extract EXIF information from bitmap. Returns false if no EXIF information
   // is embedded in the bitmap.
+  bool ExifCameraModel(std::string& camera_model) const;
```

Comment:
The convention in COLMAP is to use pointers for output arguments.

Reply:
In other words, this should be std::string*

Reply:
Fixed